app/utils/dictionary/tzAbbr.ts

```diff
@@ -0,0 +1,228 @@
+export const tzAbbr = {
+  'America/Araguaina'                : 'BRT',
+  'America/Argentina/Buenos_Aires'   : 'ART',
+  'America/Argentina/Catamarca'      : 'ART',
+  'America/Argentina/ComodRivadavia' : 'ART',
+  'America/Argentina/Cordoba'        : 'ART',
+  'America/Argentina/Jujuy'          : 'ART',
+  'America/Argentina/La_Rioja'       : 'ART',
+  'America/Argentina/Mendoza'        : 'ART',
+  'America/Argentina/Rio_Gallegos'   : 'ART',
+  'America/Argentina/Salta'          : 'ART',
+  'America/Argentina/San_Juan'       : 'ART',
+  'America/Argentina/San_Luis'       : 'ART',
+  'America/Argentina/Tucuman'        : 'ART',
+  'America/Argentina/Ushuaia'        : 'ART',
```

Comment:
Please remove any ambiguous entry, i.e., any code which is repeated for 2 timezones

Reply:
but they all are diff timezones

Reply:
Exactly, and they all show ART, which is ambiguous

Reply:
I understand all but like 
'America/Argentina/Buenos_Aires'   : 'ART',
'America/Argentina/Catamarca'      : 'ART',

both are unique so removing any on one results in not getting tz abbr

@iamareebjamal 
I want to know that is there a way we can create multiple keys with a single value object?

Reply:
No, just remove them and let the offset show as the timezone like the current situation

When you see CRT, should it be considered China Standard Time or Cuba Standard Time. Hence, remove all entries which have ambiguous abbreviations between two timezones.

Reply:
ohk like that :+1:
but also I think I should not remove that abbr where they are duplicated but have same full forms like in above ART is used for Argentina Time which is same?

Reply:
No, they can be confusing, better to remove them

Reply:
Last question like 'TRT' abbr is for 3 of them then which one to keep and which to remove?

Reply:
Remove all which have more than one, so no TRT will be shown, because then how would a person know which of the 3 TRT represents?